demos/eigenvalues_QG_basinmodes/qgbasinmodes.py.rst

```diff
@@ -154,13 +154,22 @@ We define the Test Function :math:`\phi` and the Trial Function
 To build the weak formulation of our equation we need to build two PETSc
 matrices in the form of a generalized eigenvalue problem,
 :math:`A\psi = \lambda M\psi`. We impose the boundary conditions on the
-mass matrix :math:`M`, since that is where we used integration by parts. ::
+mass matrix :math:`M`, since that is where we used integration by parts.
+For a more stable computation of the eigenvalues, we also create
```

Comment:
"stable computation" reads wrong to me. I think "To avoid spurious eigenvalues associated with the Dirichlet rows of the operator, we extract submatrices to remove them" or something like that?

Reply:
Ok, I'll change it.